.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,48 @@
- **JIRA**: (link to JIRA ticket)
- **Linked PRs**: (links to corresponding PRs, optional)
+<!--- Provide a general summary of your changes in the Title above -->
 
-## How to test-drive this PR
-- (step1)
+# Description
 
-## Changes
-- (change1)
+<!--- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
 
-## Todos
+# Types of changes
 
-### General
-- [ ] Add a high-level description of your changes to relevant CHANGELOG.md files (**not** required for doc updates)
-- [ ] The code changes are covered by test cases
-- [ ] Analytics events instrumented to measure impact of change
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 
-### Backwards Compatibility
-- [ ] This PR contains no breaking changes or breaking changes have been approved by Product Management/Engineering as necessary.
+- [ ] **Bug fix** (non-breaking change which fixes an issue)
+- [ ] **New feature** (non-breaking change which adds functionality)
+- [ ] **Documentation update**
+- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
 
 > Breaking changes include:
-> * Removing a public function/Component/Prop
-> * Adding required arguments to functions
-> * Changing the data-types of function params or return values
+>
+> - Removing a public function/Component/Prop
+> - Adding required arguments to functions
+> - Changing the data-types of function params or return values
+> - Adding new peerDependency to package.json
+
+# Changes
+
+- (change1)
+
+# How to test-drive this PR
+
+- (step1)
+
+# Checklists
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+
+## General
+
+- [ ] My code follows the code style of this project
```

Comment:
This line is a bit ambiguous imo 🤔 . What does "code style" mean in this context? Eslint? prettier? or something more?

Reply:
We could leave this out for now and wait until PRs start rolling in to see if code style even becomes an issue. If so, we could create some more specific guidelines and best practices that go beyond the ESlint and Prettier rules.

Reply:
👍